pkgs/development/python-modules/plotly/default.nix

```
@@ -70,6 +71,9 @@ buildPythonPackage rec {
     scikit-image
   ];
 
+  # the check inputs are broken on darwin
+  doCheck = !stdenv.hostPlatform.isDarwin;
+
```

```suggestion
  optional-dependencies = {
    orca = [ orca ];
  };
```

and change `nativeCheckInputs` to `checkInputs`